src/gazebo_mavlink_interface.cpp

```diff
@@ -410,9 +411,9 @@ void GazeboMavlinkInterface::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf
   gravity_W_ = world_->GetPhysicsEngine()->GetGravity();
 
   if (_sdf->HasElement("imu_rate")) {
-    imu_rate_ = 1 / _sdf->GetElement("imu_rate")->Get<int>();
-    set_imu_rate_ = true;
+    imu_rate_ = _sdf->GetElement("imu_rate")->Get<int>();
```

Comment:
Intended?

Reply:
Actually now that I see, it's not also. @arwagoner the idea is that the IMU rate is set only if we have a parameter setting it. It should by default by at max rate. So https://github.com/PX4/sitl_gazebo/pull/157/files#diff-5ed56584b86a34e30277b2fd7cb7a917R126 also doesn't make sense to be init.

Reply:
It needs to have an initial value otherwise it defaults to 0.